<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayToolClass</class>
 <widget class="QMainWindow" name="DisplayToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplayTool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.136, y1:0.210136, x2:0.965909, y2:0.989, stop:0.386364 rgba(37, 21, 130, 255), stop:0.528409 rgba(28, 19, 127, 255), stop:1 rgba(66, 21, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="4">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">border:  2px solid #4AEEF1;
background-color: rgb(71, 83, 240);
border-radius:20px;
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">border:none;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <widget class="QWidget" name="page_3">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="styleSheet">
              <string notr="true">border: none;
border-radius:20px;</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0" columnstretch="1">
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="0" rowspan="2">
               <widget class="QScrollArea" name="scrollArea">
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContents</enum>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>831</width>
                   <height>449</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QStackedWidget" name="stackedWidgetHome">
                    <widget class="QBtnListWdgt" name="TopPage" native="true"/>
                    <widget class="QBtnListWdgt" name="secondPage"/>
                    <widget class="QBtnListWdgt" name="thirdPage"/>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QvideoListTree" name="page_4"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="homeButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="BackBtn">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>上一级</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QvideoListTree</class>
   <extends>QWidget</extends>
   <header location="global">qvideolisttree.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QBtnListWdgt</class>
   <extends>QWidget</extends>
   <header location="global">qbtnlistwdgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="displaytool.qrc"/>
 </resources>
 <connections/>
</ui>
